src/index.html

```diff
@@ -5,9 +5,13 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Wbpack Exercise</title>
+  <script src="https://kit.fontawesome.com/242e63f88a.js" crossorigin="anonymous"></script>
 </head>
 
 <body>
+  <!-- fontawesome icons how to use:
+  Heart icon: <i class="fa-regular fa-heart"></i>
+  X icon: <i class="fa-sharp fa-solid fa-xmark"></i> || <i class="fa-solid fa-xmark"></i> -->
```

Comment:
👍🏿 Nice work so far but i have some suggestions to give.
I noticed that you have some commented-out code in your codebase. I wanted to let you know that leaving commented code in your codebase can make your code difficult to read and understand. It can also cause confusion for other developers who may work on your code in the future. To avoid these issues, it's always better to remove any commented-out code from your codebase as soon as possible. This will help keep your codebase clean, organized, and easy to maintain. I hope this feedback is helpful, and please don't hesitate to reach out if you have any questions or concerns.  😉